src/realm/table.cpp

```diff
@@ -2493,7 +2494,7 @@ ObjKey Table::find_first_decimal(ColKey col_key, Decimal128 value) const
 
 ObjKey Table::find_first_string(ColKey col_key, StringData value) const
 {
-    return find_first(col_key, value);
+    return find_first<StringData>(col_key, value);
```

Comment:
what is the purpose of this change? since `value` is a `StringData` the correct overload for `find_first` should be deduced implicitly.

Reply:
Ok I will remove it and see if it still works.